lib/query_builder/schema/comparison_filter.ex

```diff
@@ -38,10 +38,18 @@ defmodule EctoShorts.QueryBuilder.Schema.ComparisonFilter do
     where(query, [scm], is_nil(field(scm, ^filter_field)))
   end
 
+  defp build_subfield_filter(query, filter_field, :==, val) do
+    where(query, [scm], field(scm, ^filter_field) == ^val)
+  end
+
   defp build_subfield_filter(query, filter_field, :!=, nil) do
     where(query, [scm], not is_nil(field(scm, ^filter_field)))
   end
 
+  defp build_subfield_filter(query, filter_field, :!=, val) do
```

Comment:
Great addition!

Reply:
Thanks, @MikaAK !! I really appreciate it :)